get-started.md

```diff
@@ -9,8 +9,7 @@ This guide uses Docker to run all the components of LangStream locally. If you w
 1. Install the LangStream CLI:
 
 ```
-brew tap LangStream/langstream
```

Comment:
what's the difference ?

the new syntax handle upgrades ? 

currently I am using this flow:

```
brew update
brew upgrade langstream
```

Reply:
this one is one-liner and it automatically handles updates.. 

I think it's compatible from the previous command